docs/bigquery/snippets.py

```diff
@@ -1007,71 +1011,75 @@ def test_load_table_from_uri_parquet(client, to_delete):
     # [END bigquery_load_table_gcs_parquet]
 
 
-def test_load_table_from_uri_csv_autodetect(client, to_delete):
+def test_load_table_from_uri_autodetect(client, to_delete):
+    """Different file formats should have their own tested load sample
+    (test_load_table_from_uri_format), but repetitive code should be reduced
```

Comment:
Nit: First line should be a one-line description of the sample.

> Snippet to load with autodetected schema (all supported formats).

Also, since you have reduced repetitive code, "should" isn't quite the right word. It reads too passive.

How about:

> Each file format has it's own tested load from URI sample. Since most of the code is common for autodetect, append, and truncate, this sample includes snippets for all supported formats but only calls a single load job.

Reply:
Updated